habibaudu/Store-Manager
# #161531803 creates A User account

#### What does this PR do?
Creates A Stores attendants user Account

#### Description of Task to be completed?
Creates a users table
Creates  auth.js file to verify Token
creates utility folder, adds a helper.js file to veryToken, generateToken, Hashpassword, comparepassword
creates User controller to creates a user

#### How should this be manually tested?
go to postman enter: localhost:8080/api/v1/users
enter username,password, email and Role click send
should see : 'Registration was successful' 
#### What are the relevant pivotal tracker stories?

https://www.pivotaltracker.com/story/show/161531803

#### Screenshots
![pr1](https://user-images.githubusercontent.com/29799658/47635691-37860300-db56-11e8-9986-f8e5fa87f524.PNG)


 #161531803
